ui/src/app/navigation/commoncontrols.component.ts

```diff
@@ -14,7 +14,10 @@ export class CommoncontrolsComponent extends AbstractSearchComponent implements
   searchIcon = SvgHelper.path(SvgIcon.SEARCH)
   dismissIcon = SvgHelper.path(SvgIcon.DISMISS)
 
-  constructor(protected searchService: SearchService, protected route: ActivatedRoute, protected authService: AuthService) {
```

Comment:
It doesn't look like you made any functional changes in this file. Can you please revert the whitespace changes?